imranariffin/small-steps-api
# Release 0.0.1 (#36)

* Enhance/3/deploy dev environment (#13)

* Ensure db migrate on each release

* Use postgres as database

* Introduce new env var `ENV`

* Inherit require env vars from parent environment

* Remove hard-coded credentials

* Setup test environment variables

* Create first test case

* Configure test environment in heroku app.json

* Use ENV=test-ci to separate local and ci test

* Move tests from api to app folder

* Create goal on POST request. Always use UTC for date

* Return 405 on unsupported http methods

* Support goals list

* Add missing library `freezegun` into requirements.txt

* Print verbose & expect no input during test

* Unrequire `ENV` env variable for testing

* Enable delete goals on /v1/goals/<id>/

* Mark goal as deleted instead of actual deletion

* Init account create

* Return error when device id already exists

* Support only POST method for now

* Support only json content-type
